docs/sources/send-data/lambda-promtail/_index.md

```diff
@@ -105,25 +105,49 @@ This workflow allows ingesting AWS VPC Flow logs from s3.
 
 One thing to be aware of with this is that the default flow log format doesn't have a timestamp, so the log timestamp will be set to the time the lambda starts processing the log file.
 
+If using CloudFormation to write your infrastructure code, you should consider the [EventBridge based solution](#s3-based-logging-and-cloudformation) for easier deployment.
+
 ### Loadbalancer logs
 
 This workflow allows ingesting AWS Application/Network Load Balancer logs stored on S3 to Loki.
 
+If using CloudFormation to write your infrastructure code, you should consider the [EventBridge based solution](#s3-based-logging-and-cloudformation) for easier deployment.
+
 ### Cloudtrail logs
 
 This workflow allows ingesting AWS Cloudtrail logs stored on S3 to Loki.
 
+If using CloudFormation to write your infrastructure code, you should consider the [EventBridge based solution](#s3-based-logging-and-cloudformation) for easier deployment.
+
 ### Cloudfront logs
 Cloudfront logs can be either batched or streamed in real time to Loki:
 + Logging can be activated on a Cloudfront distribution with an S3 bucket as the destination. In this case, the workflow is the same as for other services (VPC Flow logs, Loadbalancer logs, Cloudtrail logs).
 + Cloudfront [real-time logs](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/real-time-logs.html) can be sent to a Kinesis data stream. The data stream can be mapped to be an [event source](https://docs.aws.amazon.com/lambda/latest/dg/invocation-eventsourcemapping.html) for lambda-promtail to deliver the logs to Loki.
 
+If using CloudFormation to write your infrastructure code, you should consider the [EventBridge based solution](#s3-based-logging-and-cloudformation) for easier deployment.
```

Comment:
I feel like saying this four times is repetitive.  Could we maybe just say it once, up in the Examples, CloudFormation section, right after line 86?

Reply:
Agree, moved there